android/gradle.properties

```diff
@@ -11,7 +11,7 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryErro
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 #Tue Dec 19 15:08:27 EST 2023
-KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=4.0.0
+KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=rel~4.0.1-SNAPSHOT
```

Comment:
Won't merge till 4.0.1 is actually released and will update to the official build.

Reply:
```suggestion
KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=4.0.1
```